azure/packages/azure-service-utils/api-report/azure-service-utils.legacy.alpha.api.md

```diff
@@ -11,6 +11,4 @@ export { IUser }
 
 export { ScopeType }
 
-// (No @packageDocumentation comment for this package)
```

Comment:
Do you know why this went away?

Reply:
Jason added support for retaining the `@packageDocumentation` comments with our custom rollup generation.